### PR TITLE
[zh-cn] Localize kubectl_top/_index.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+<!--
+Display resource (CPU/memory) usage.
+
+ The top command allows you to see the resource consumption for nodes or pods.
+
+ This command requires Metrics Server to be correctly configured and working on the server.
+-->
+显示资源（CPU/内存）使用情况。
+
+- top 命令允许您查看节点或 Pod 的资源消耗情况。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:26]:
```suggestion
- top 命令允许你查看节点或 Pod 的资源消耗情况。
```
可以全文替换 您

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 2]
```diff
@@ -0,0 +1,653 @@
...
+
+- 此命令要求 Metrics Server 在服务器上正确配置并运行。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:28]:
```suggestion
- 此命令要求 Metrics Server 在服务器上被正确配置且正常运行。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 3]
```diff
@@ -0,0 +1,653 @@
...
+
+```
+kubectl top [flags]
+```
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:32]:
```suggestion
```shell
kubectl top [flags]
```
```
make it prettier to add a `shell`

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 4]
```diff
@@ -0,0 +1,653 @@
...
+<tr>
+<td colspan="2">--categories strings</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Limit to resources that belong to the specified categories.
+-->
+指定资源的类别。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:107]:
```suggestion
筛选属于指定类别的资源。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 5]
```diff
@@ -0,0 +1,653 @@
...
+<tr>
+<td colspan="2">--namespaced&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default: true-->默认值：true</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+If false, non-namespaced resources will be returned, otherwise returning namespaced resources by default.
+-->
+如果为false，则返回非命名空间作用域的资源，否则默认返回命名空间作用域的资源。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:135]:
```suggestion
如果为 false，则返回非命名空间作用域的资源，否则默认返回命名空间作用域的资源。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 6]
```diff
@@ -0,0 +1,653 @@
...
+<tr>
+<td colspan="2">--api-group string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Limit to resources in the specified API group.
+-->
+限制为指定 API 组中的资源。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:79]:
```suggestion
筛选指定 API 组中的资源。
```
This page has 3 `Limit to ...`, we can use a consistent style.

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md [hunk 7]
```diff
@@ -0,0 +1,653 @@
...
+## {{% heading "seealso" %}}
+
+<!--
+* [kubectl](../kubectl/)	 - kubectl controls the Kubernetes cluster manager
+* [kubectl top node](kubectl_top_node/)	 - Display resource (CPU/memory) usage of nodes
+* [kubectl top pod](kubectl_top_pod/)	 - Display resource (CPU/memory) usage of pods
+-->
+* [kubectl](../kubectl/) - kubectl 控制 Kubernetes 集群管理器
+* [kubectl top node](kubectl_top_node/)	- 显示节点的资源（CPU/内存）使用情况
+* [kubectl top pod](kubectl_top_pod/)	- 显示 Pod 的资源（CPU/内存）使用情况
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_top/_index.md:653]:
```suggestion
* [kubectl top pod](kubectl_top_pod/) - 显示 Pod 的资源（CPU/内存）使用情况
```

**Review by ydFu — APPROVED** [review]
/lgtm
/approve

**Pull request merged**